docs/TESTS.md

```diff
@@ -112,3 +112,10 @@ unusable for further testing, typically by hardening it.
 
 A test that just installs extra RPMs from the package manager, or enables
 extra services, is not considered destructive.
+
+## `subset-profile`
+
+A tests for a profile which is a subset of other profile test.
+
+A subset profile's ruleset is covered by the superset. However, profile
+variables in the superset profile might be different.
```

Comment:
I would personally like a bit more background for this. Feel free to copy/paste from this:

```
Some compliance standards specify multiple "levels" of system hardening,
with the more permissive levels typically being a subset of the more
restrictive ones, ie. Minimal, Intermediary and Enhanced.

This tag signifies that a test covers one of the levels (profiles), which
are themselves a subset of another level (profile).

There are other minor factors (content profile variables) that break this
relationship, however, if you want a smaller test set at the cost of ignoring
these factors, use this tag.
```
It would personally help me when I'm staring at this in 4 years, wondering why the tag exists. :smile: